.semaphore/live-site-deploy.yml

```diff
@@ -20,5 +20,5 @@ blocks:
             - npm install
             - gem install bundler
             - bundle install
-            - JEKYLL_ENV=production bundle exec jekyll build
-            - aws s3 cp --recursive ./_site s3://kafka-tutorials.confluent.io/
+            - JEKYLL_ENV=production bundle exec jekyll build --baseurl /tutorials
+            - aws s3 cp --recursive ./_site s3://kafka-tutorials.confluent.io/tutorials
```

Comment:
Looking at this, looks like you guys are uploading the built site to `kafka-tutorials.confluent.io/tutorials`.

Am I correct to assume that whatever is already in S3 on the root, will continue to stay there? For example, if there is a file in `kafka-tutorials.confluent.io/robots.txt`, it will stay there?

Reply:
@hartantothio cc: @bbejeck Due to https://github.com/confluentinc/kafka-tutorials/issues/921, existing assets do not get deleted from S3.  As for updating it, the new builds would all put `robots.txt` at kafka-tutorials.confluent.io/robots.txt (see http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/tutorials/robots.txt as evidence).  Is this a problem?